src/Microsoft.Extensions.Configuration.Xml/EncryptedXmlDocumentDecryptor.cs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-#if !DOTNET && !DNXCORE50
+#if !DOTNET5_4 && !DOTNET5_4
```

Comment:
hey man @divega , is there any documentation for those dotnet "generations" changes? I just took a quick look on the DNX repo but couldn't find anything. Just wondering what all that actually means / translates to ~~